test/unit_tests/braket/aws/test_aws_device.py

```diff
@@ -501,6 +531,36 @@ def test_run_no_extra(aws_quantum_task_mock, device, circuit):
     )
 
 
+@pytest.mark.xfail(raises=ValueError)
```

Comment:
Where are we actually testing the passing version of this?

Reply:
Right now execution with free parameters is blocked. In a subsequent PR, the execution will be enabled.